storage-quickstart.go

```diff
@@ -32,114 +29,101 @@ func randomString() string {
 	return strconv.Itoa(r.Int())
 }
 
-func handleErrors(err error) {
-	if err != nil {
-		if serr, ok := err.(azblob.StorageError); ok { // This error is a Service-specific
-			switch serr.ServiceCode() { // Compare serviceCode to ServiceCodeXxx constants
-			case azblob.ServiceCodeContainerAlreadyExists:
-				fmt.Println("Received 409. Container already exists")
-				return
-			}
-		}
-		log.Fatal(err)
-	}
-}
-
 func main() {
 	fmt.Printf("Azure Blob storage quick start sample\n")
 
-	// From the Azure portal, get your storage account name and key and set environment variables.
-	accountName, accountKey := os.Getenv("AZURE_STORAGE_ACCOUNT"), os.Getenv("AZURE_STORAGE_ACCESS_KEY")
-	if len(accountName) == 0 || len(accountKey) == 0 {
-		log.Fatal("Either the AZURE_STORAGE_ACCOUNT or AZURE_STORAGE_ACCESS_KEY environment variable is not set")
-	}
+	// Replace `<StorageAccountName` with your storage account
+	url := "https://storageblobsgo.blob.core.windows.net/" //replace with <StorageAccountName>
```

Comment:
Should we do something like this which is guaranteed to not exist?
```suggestion
	url := "https://TODO_REPLACE_WITH_STORAGEACCOUNTNAME.blob.core.windows.net/" //replace with <StorageAccountName>
```